pkg/dns/azure/dns_test.go

```diff
@@ -44,49 +44,76 @@ func TestEnsureDNS(t *testing.T) {
 		ID: "/subscriptions/E540B02D-5CCE-4D47-A13B-EB05A19D696E/resourceGroups/test-rg/providers/Microsoft.Network/dnszones/dnszone.io",
 	}
 	err = mgr.Ensure(&record, dnsZone)
+	t.Log("Ensure with single address")
 	if err != nil {
-		t.Fatal("failed to ensure dns")
+		t.Fatalf("unexpected error from Ensure: %v", err)
 		return
 	}
 
 	recordedCall, _ := fc.RecordedCall(rg, zone, ARecordName)
 
 	if recordedCall != "PUT" {
-		t.Fatalf("expected the dns client 'Delete' func to be called, but found %s instead", recordedCall)
+		t.Fatalf("expected the dns client 'Ensure' func to be called, but found %s instead", recordedCall)
```

Comment:
Change `%s` → `%q`.  Sorry I missed this before.

Reply:
done